pkg/cassandra/config/config.go

```diff
@@ -45,6 +45,8 @@ type Configuration struct {
 	DisableAutoDiscovery bool          `yaml:"disable_auto_discovery"`
 	EnableDependenciesV2 bool          `yaml:"enable_dependencies_v2"`
 	TLS                  TLS
+	TagIndexWhitelist    []string `yaml:"tag-index-whitelist"`
+	TagIndexBlacklist    []string `yaml:"tag-index-blacklist"`
```

Comment:
this seems like the wrong place for these options (and `enable_dependencies_v2`, for that matter), because this struct is meant as a generic Cassandra connection configuration, not the SpanWriter configuration.

Reply:
There does not appear to be a generic SpanWriter configuration and these settings are Cassandra specific.  I could make a `SpanWriterConfiguration` struct in the same package?

Reply:
Ignore.  Adding it to the cassandra Options struct like `SpanStoreWriteCacheTTL`